README.md

```diff
@@ -31,6 +31,30 @@ conda build recipe -c conda-forge --output-folder dist/conda --no-anaconda-uploa
 conda install -c conda-forge -c ./dist/conda/ coiled-runtime
 ```
 
+## Test
+
+The `coiled-runtime` test suite can be run locally with the following steps:
+
+1. Ensure your local machine is authenticated to use the `dask-engineering` Coiled account and
+   the Coiled Dask Engineering AWS S3 account.
+2. Create a Python environment and install development dependencies: `pytest`, `jinja2`, and `packaging`
```

Comment:
`conda` is also a dependency, unless it's `importorskip`ped in `test_build.py`.

Reply:
Thanks -- included `conda` in the latest commit